lib/cimp/repo.go

```diff
@@ -0,0 +1,72 @@
+package cimp
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/consul/api"
+)
+
+type Config struct {
+	Address string
+}
+
+type ConsulRepo struct {
+	client *api.Client
+}
+
+func InitRepo(cfg Config) (*ConsulRepo, error) {
```

Comment:
mb Repo -> Storage? it's would be more natural for e.g consulStorage

Reply:
valid

Reply:
> mb Repo -> Storage? it's would be more natural for e.g consulStorage

fixed